roles/st2-web/tasks/main.yml

```diff
@@ -0,0 +1,70 @@
+---
+# tasks file for st2-web
+
+
+
+- name: Install nginx
+  sudo: yes
+  apt:
+    name: nginx
+    state: present
+
+- name: Install latest st2-web package
+  sudo: yes
+  apt:
+    name: st2web
+    state: latest
+  when: st2_version == "stable"
+  notify:
+    - restart st2
+
+- name: Install exact st2-web package
+  sudo: yes
+  apt:
+    name: st2web={{ st2_version }}.{{ st2_revision }}
```

Comment:
`st2web` has independent revision number.
Eg. revision number for `st2` could be `150`, while for `st2web` latest revision number could be `6`.